### PR TITLE
gic_v3: fix EOImode bit number

### PR DESCRIPTION
**Pull request by JorgeMVP**
Signed-off-by: Jorge Pereira <jorgepereira89@gmail.com>

### CODE REVIEW

File: src/arch/arm/machine/gic_v3.c [hunk 1]
```diff
@@ -277,7 +277,7 @@ BOOT_CODE static void cpu_iface_init(void)
 
     /* EOI drops priority and deactivates the interrupt: ICC_CTLR_EL1 */
     SYSTEM_READ_WORD(ICC_CTLR_EL1, icc_ctlr);
-    icc_ctlr &= ~BIT(GICC_CTLR_EL1_EOImode_drop);
```
Comment (yyshen) [inline src/arch/arm/machine/gic_v3.c:280]: it looks like the EOImodeNS bit in bit 9. please double check.

Comment (JorgeMVP) [reply src/arch/arm/machine/gic_v3.c:280]: Based on this: https://developer.arm.com/documentation/ddi0595/2021-03/AArch64-Registers/ICC-CTLR-EL1--Interrupt-Controller-Control-Register--EL1-

EOImode bit is the bit 1 of the ICC_CTLR_EL1, Interrupt Controller Control Register (EL1)

Comment (JorgeMVP) [reply src/arch/arm/machine/gic_v3.c:280]: Section 12.2.6 ICC_CTLR_EL1, Interrupt Controller Control Register (EL1) of GICv3 spec

Comment (kent-mcleod) [reply src/arch/arm/machine/gic_v3.c:280]: @yyshen, It's bit 9 in the MMIO interface (GICC_CTLR), but bit 1 in the system register interface (ICC_CTLR).

**Comment by axel-h** [comment]
@yyshen: Looking at https://github.com/seL4/seL4/blob/e4262a90d2828149e0bf1a944afd2e77da3bb090/include/arch/arm/arch/machine/gic_v3.h#L48
ths PR look fine to fix the described bug. It there are more issues beyond that, this should become a separate issue.

**Review by axel-h — APPROVED** [review]

**Review by kent-mcleod — APPROVED** [review]
Nice fix.

**Comment by axel-h** [comment]
@JorgeMVP could you rebase this to allow merging, seems there is some conflict that can't be resolved automatically.

**Comment by JorgeMVP** [comment]
@axel-h done. Thanks

**Review by yyshen — APPROVED** [review]

**Pull request merged**